beokay.py

```diff
@@ -68,7 +68,8 @@ def ensure_paths(parsed_args):
 
 
 def git_clone(repo, branch, path):
-    subprocess.check_call(["git", "clone", repo, path, "--branch", branch])
+    subprocess.check_call(["git", "clone", repo, path, "--branch", branch],
+                          env=os.environ)
```

Comment:
I thought this was done by default?

Reply:
Looks like you are correct - jumped the gun on this one.